.github/workflows/close-stale-prs.yml

```diff
@@ -19,10 +19,18 @@ jobs:
           # appropriately.
           github-token: ${{ secrets.PROJEN_GITHUB_TOKEN }}
           stale-days: 21
-          response-days: 7
+          response-days: 14
 
           # Optional
           important-checks-regex: AutoBuildv2Project1C6BFA3F
+          merge-conflict-warning: |
+            This PR cannot be merged because it has conflicts. Please resolve them. Note that PRs with merge conflicts considered stale and will be closed if not resolved.
+
+            To prevent automatic closure:
+            - Fix merge conflicts
+            - OR request an exemption by adding a comment containing 'Exemption Request' with justification e.x "Exemption Request: <justification>"
```

Comment:
After they request exemption and we review it, what label do we need to give to not prevent it to close? `pr/do-not-close`? This doesn't quite feel the same as the typical flow with exemption request.

Reply:
Yea the maintainer would have to put `do-not-close` or leave the exemption label there. If they remove the label and don't add the `do-not-close` label then the PR could be considered stale.